react/src/pages/StartPage.tsx

```diff
@@ -17,16 +19,27 @@ import { useTranslation } from 'react-i18next';
 const StartPage: React.FC = () => {
   const { t } = useTranslation();
 
-  // to avoid flickering
-  useSuspendedBackendaiClient();
+  const baiClient = useSuspendedBackendaiClient();
+  const blockList = (baiClient?._config?.blockList as MenuKeys[]) ?? [];
+  const inactiveList = (baiClient?._config?.inactiveList as MenuKeys[]) ?? [];
```

Comment:
It would be better to specify the type of _.config?.blockList as MenuKeys rather than forcing a cast to as.